src/overlays/actors/ovl_Obj_Um/z_obj_um.c

```diff
@@ -14,15 +14,15 @@
 
 /**
  * weekEventReg flags checked by this actor:
- * - WEEKEVENTREG_22_01: Aliens defeated
+ * - WEEKEVENTREG_DEFENDED_AGAINST_THEM: Aliens defeated
```

Comment:
Also `THEM` is even harder because you lose the capitalization.
From a quick search, seems like "Aliens" is more common in the codebase. "Them" is only used in 2 functions of EnMa4 (Romani)